src/test/tools/pmemobjcli/shortencommands.py

```diff
@@ -0,0 +1,148 @@
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+
+import sys
+import re
+import os.path
+
+
+def validate_input():
+    if len(sys.argv) > 1:
+        if not os.path.isfile(sys.argv[1]):
+            print("Error: pmemobjcli file %s does not exist." % sys.argv[1])
+            return False
+        if len(sys.argv) > 2:
+            if not os.path.isfile(sys.argv[2]):
+                print("Error: input file %s does not exist." % sys.argv[2])
+                return False
+            if len(sys.argv) > 4:
+                for index in range(4, len(sys.argv)):
+                    print("Warning: Ignoring extra parameter: " + sys.argv[index])
+        else:
+            print("Error: Input file path empty. Please enter a correct path to input file.")
+            return False
+    else:
+        print_help()
+        return False
+
+    return True
+
+
+def pmem_obj_cli_functions_block():
+    lines = []
+    pmem_obj_cli_file = sys.argv[1]
+    pattern_of_beginning = re.compile("(pocli_cmd pocli_commands)")
+    pattern_of_end = re.compile("(};)")
+    start_appending = False
+    start_of_command_block = -1
+    end_of_command_block = -1
+    for i, line in enumerate(open(pmem_obj_cli_file)):
+        for _ in re.finditer(pattern_of_beginning, line):
+            start_of_command_block = i
+            start_appending = True
+        if start_appending:
+            lines.append(line)
+            for _ in re.finditer(pattern_of_end, line):
+                end_of_command_block = i
+        if start_of_command_block > 0 and end_of_command_block > 0:
+            break
+    return lines
+
+
+def shortened_command_names(lines):
+    dictionary = {}
+    pattern = re.compile("(^(?!static struct pocli_cmd pocli_commands))(.*{.*)")
+
+    pattern2 = re.compile("\",.*")
+    for i, line in enumerate(lines):
+        for match in re.finditer(pattern, line):
+            long_name = lines[i+1].strip()
+            long_name = re.sub(pattern2, "", long_name)
+            long_name = long_name.replace("\"", "")
+
+            short_name = lines[i + 2].strip()
+            short_name = re.sub(pattern2, "", short_name)
+            short_name = short_name.replace("\"", "")
+
+            dictionary[long_name] = short_name
+
+    return dictionary
+
+
+def parse_input_file(dictionary):
+    input_file = sys.argv[2]
+    lines = []
+    for i, line in enumerate(open(input_file)):
+        splited_line = line.split()
+        if len(splited_line) > 0:
+            word = splited_line[0]
+            if word in dictionary:
+                line = line.replace(word, dictionary[word])
+        lines.append(line)
+    return lines
+
+
+def write_lines_to_output(lines, output_file_path):
+    output_file = open(output_file_path, 'w')
+    output_file.writelines(lines)
+    output_file.close()
+    print("Output written to : " + os.path.abspath(output_file.name))
+
+
+def print_help():
```

Comment:
please use a dedicated parser for arguments, https://docs.python.org/3/library/argparse.html